packages/blocks/src/page-block/utils/container-operations.ts

```diff
@@ -133,7 +132,6 @@ export function handleBackspace(page: Page, e: KeyboardEvent) {
       const { quill } = richText;
       if (isCollapsedAtBlockStart(quill)) {
         // use quill handler
-        noop();
```

Comment:
This line should be kept. This is an explicit `noop`, telling people we intentionally do nothing here.